src/modules/ekf2/ekf2_main.cpp

```diff
@@ -484,6 +484,25 @@ void Ekf2::task_main()
 			ctrl_state.pitch_rate = _lp_pitch_rate.apply(sensors.gyro_rad_s[1]);
 			ctrl_state.yaw_rate = _lp_yaw_rate.apply(sensors.gyro_rad_s[2]);
 
+			/* Velocity in Body Frame */
+			float velocity[3];
+			_ekf->copy_velocity(velocity);
+			Vector3f v_n(velocity);
+			matrix::Dcm<float> Tnb(q);
+			Vector3f v_b = Tnb * v_n;
```

Comment:
the idea was to implement it the same way it is done in [ekf_att_pos_estimator](https://github.com/PX4/Firmware/blob/43e0186eb383bbbbd13fb9afbec95530d0fb643f/src/modules/ekf_att_pos_estimator/ekf_att_pos_estimator_main.cpp#L878:L880)